app/src/main/java/chat/rocket/android/privacy/ui/PrivacyFragment.kt

```diff
@@ -65,6 +66,13 @@ class PrivacyFragment : Fragment(), PrivacyView {
     }
 
     override fun showDiscoverability(discoverability: String) {
+        with (activity as MainActivity) {
+            everyone = getString(R.string.msg_privacy_option_everyone)
+            contacts = getString(R.string.msg_privacy_option_contacts)
+            none = getString(R.string.msg_privacy_option_none)
+        }
+        val values = arrayListOf("all", "contacts", "none")
+        val choices = arrayListOf(everyone, contacts, none)
```

Comment:
val choices = arrayListOf(
activity.getString(R.string.msg_privacy_option_everyone),
activity.getString(R.string.msg_privacy_option_contacts), 
activity.getString(R.string.msg_privacy_option_none))

We can directly do this?

Reply:
That's nice.......  @catmang can you test this implementation, and if it works, it does seem more concise...  Thanks @Shailesh351